### PR TITLE
Implemented the ProductImage Model

### PR DESCRIPTION
**Pull request by AhmadAlBarasy**

### CODE REVIEW

File: src/models/ProdctImage.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import sequelize from "../database";
+import { DataTypes, Model } from "sequelize";
+
+class ProductImage extends Model {}
+
+ProductImage.init(
+    {
+        id: {
+            type: DataTypes.INTEGER,
```
Comment (MahmoudMDarabaih) [inline src/models/ProdctImage.ts]: Change to UUID

File: src/models/ProdctImage.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+            autoIncrement: true,
```
Comment (MahmoudMDarabaih) [inline src/models/ProdctImage.ts]: add  defaultValue: DataTypes.UUIDV4,

**Review by MahmoudMDarabaih — CHANGES_REQUESTED** [review]

**Comment by AhmadAlBarasy** [comment]
Please review the changes

**Review by MahmoudMDarabaih — APPROVED** [review]
معللم,

**Pull request merged**